source/macos/AdaptiveCards/AdaptiveCards/Views/ACRView.swift

```diff
@@ -99,24 +99,26 @@ class ACRView: ACRColumnView {
         return subview(with: identifier, in: self)
     }
     
-    private func submitCardInputs(actionView: NSView, dataJSON: String?) {
+    private func submitCardInputs(actionView: NSView, dataJSON: String?, associatedInputs: Bool) {
         var dict = [String: Any]()
         
         // recursively fetch input handlers dictionary from the parent
         var rootView = self
         var parentView: ACRView? = self
-        repeat {
-            if let handlers = parentView?.inputHandlers {
-                for handler in handlers {
-                    guard handler.isValid else { continue }
-                    dict[handler.key] = handler.value
+        if (!renderConfig.supportsSchemeV1_3) || (renderConfig.supportsSchemeV1_3 && associatedInputs) {
```

Comment:
Would prefer to extract this condn to a variable and try to solve this using `guard..else`

Reply:
We still need to submit dataJson even if associatedInput is none